Library/Services/TopicService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * This file is part of the Disturb package.
+ *
+ * (c) Matthieu Ventura <mventura@voyageprive.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Vpg\Disturb\Services;
+
+class TopicService
+{
+    /**
+     * @const String TOPIC_WORKFLOW_MANAGER_NAME
+     */
+    const TOPIC_WORKFLOW_MANAGER_NAME = 'disturb-@workflow_name@-manager';
+
+    /**
+     * @const String TOPIC_WORKFLOW_MANAGER_STEP_NAME
+     */
+    const TOPIC_WORKFLOW_MANAGER_STEP_NAME = 'disturb-@workflow_name@-@step_name@-step';
+
+    /**
+     * Generate formated topic name for workflow manager
+     *
+     * Example : disturb-LOAD_CONTRACT_28_FR-manager
+     *
+     * @param String $workflowName Workflow Name
+     *
+     * @throws \Exception Required parameters
+     * @return String
+     */
+    public static function getWorkflowManagerTopicName (string $workflowName) : string
```

Comment:
not sure we need `Workflow` in the func name

Reply:
#30